lib/aws.ts

```diff
@@ -0,0 +1,33 @@
+import AWS, { AWSError } from 'aws-sdk';
+import { BadRequestError } from './utils/errors/badRequestError';
+
+const { AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY } = process.env;
+const AWS_REGION = 'us-east-2';
```

Comment:
⛏️ These ought to be env variables

Reply:
Or, at least configurable via environemnt variables:
```suggestion
const AWS_REGION = process.env.AWS_REGION ?? 'us-east-2';
```